### PR TITLE
First version of "import" page is ready

### LINKED ISSUE #17: Walkthrough - Import
- [x] Import images
- [x] Import models
- [x] Import segmentation and transcription

Use https://lectaurep.hypotheses.org/documentation/escriptorium-tutorial-en as a basis

### PR DESCRIPTION
**Pull request by alix-tz**
Some "todo" remaining: left for later in #24 #25 and #26 

Fixes #17

### CODE REVIEW

**Review by HugoSchtr — CHANGES_REQUESTED** [review]
It looks very good, good job! I've only made a few comments on phrasing and references!

**Comment by HugoSchtr** [comment]
Well, I have requested changes, but I didn't know that it would block merging. :sweat_smile: Of course, no changes are required, my comments were only suggestions. :+1:

**Comment by FloChiff** [comment]
On top of the comments by Hugo that I also mostly agree to (I am not sure about the place of the mapping part), here are the typos I saw in the document : 
- l. 42 --> I think the correct term is "manifest" and not "manifesto" ; I also think that "metadata" is plural and not singular so it should be "are visible" instead of "is visible"
- l. 51 --> I think it should be a present "can" and not a conditional "could" as you also already used "can" right before
- l. 79 --> it should be "Known" and not "Knwon"
- l. 83 --> "useful" only has one 'l' at the end and "targeting" is written with only one 't' at the end

Also, around the lines 25-30 about the import of images, I have a recommendation for the tip related to a problem I sometimes encountered but you don't have to add it if you don't find it relevant --> when I upload big load of images (like 100 at the same time), it can make eScriptorium bug sometimes and it will not be able to upload one or two images in the mix, even though it load the images before and after. This is not a big issue as you can upload them afterwards but when you care about image orders, this is frustrating. So the advice regarding that would be to upload by little chunk (25 by 25 or something like that).

**Review by FloChiff — COMMENTED** [review]
On top of the comments by Hugo that I also mostly agree to (I am not sure about the place of the mapping part), here are the typos I saw in the document :

    l. 42 --> I think the correct term is "manifest" and not "manifesto" ; I also think that "metadata" is plural and not singular so it should be "are visible" instead of "is visible"
    l. 51 --> I think it should be a present "can" and not a conditional "could" as you also already used "can" right before
    l. 79 --> it should be "Known" and not "Knwon"
    l. 83 --> "useful" only has one 'l' at the end and "targeting" is written with only one 't' at the end

Also, around the lines 25-30 about the import of images, I have a recommendation for the tip related to a problem I sometimes encountered but you don't have to add it if you don't find it relevant --> when I upload big load of images (like 100 at the same time), it can make eScriptorium bug sometimes and it will not be able to upload one or two images in the mix, even though it load the images before and after. This is not a big issue as you can upload them afterwards but when you care about image orders, this is frustrating. So the advice regarding that would be to upload by little chunk (25 by 25 or something like that).

**Comment by alix-tz** [comment]
Thank you both for your feedback! They are all very good remarks, I'll make the changes

File: docs/walkthrough_import.md [hunk 1]
```diff
@@ -8,11 +8,75 @@ date: 2023-01-31
...
+- Check the "override" option, which will have the existing segmentation removed and replaced with the imported segmentation. If unchecked, the segmentation imported from XML files will be added to the existing segmentation.
+- Select the file (one at a time) to import from the file browser. This file can be a single ALTO XML or PAGE XML file, as well as a ZIP file containing several XML files.
+
+![image: Settings for the XML import feature](img/escriptorium_import_xml_popup.png)
+
+!!! Note
+    It is not necessary to select the pages which will be affected by the segmentation import: the mapping is automatically calculated based on the image file name and the information contained in the XML files.  
+
+!!! Tip  
+    When importing XML files generated with other tools such as Transkribus, it can be usefull to run a segmentation task targetting only the line masks <!-- todo: add link to mask recalculation --> or to preprocess the file with a tool such as [LSS](https://github.com/ponteineptique/lss).[^lss]
```
[the remark this answers is not in the document]
Comment (alix-tz) [reply docs/walkthrough_import.md:83]: yes, I think it's a good idea to point to external resources

**Comment by alix-tz** [comment]
> Also, around the lines 25-30 about the import of images, I have a recommendation for the tip related to a problem I sometimes encountered but you don't have to add it if you don't find it relevant --> when I upload big load of images (like 100 at the same time), it can make eScriptorium bug sometimes and it will not be able to upload one or two images in the mix, even though it load the images before and after. This is not a big issue as you can upload them afterwards but when you care about image orders, this is frustrating. So the advice regarding that would be to upload by little chunk (25 by 25 or something like that).

@FloChiff : you know that you can reorder images by dragging the "image" card in `/images`?

**Comment by FloChiff** [comment]
> 

@alix-tz Well I do now !!! I discover a new thing everyday about eSc thanks to this tutorial 😄 !!!

**Comment by alix-tz** [comment]
I had to rebase because the config in mkdocs.yml was not up-to-date (the rendering of admonitions was not good).

**Comment by alix-tz** [comment]
> Well, I have requested changes, but I didn't know that it would block merging. 😅 Of course, no changes are required, my comments were only suggestions. 👍

@HugoSchtr I don't understand why it's still requesting the validation of the reviews...

**Review by HugoSchtr — APPROVED** [review]
😎

**Review by FloChiff — APPROVED** [review]

**Pull request merged**